### PR TITLE
chore: setup vscode workspace

### PR DESCRIPTION
**Pull request by fargito**

### CODE REVIEW

**Review by adriencaccia — CHANGES_REQUESTED** [review]
Nice !
There is one last thing to do: add the common `.vscode` folder to `commonConfiguration` and symlink it to each package

File: commonConfiguration/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        "functions/${input:functionName}/handler.mock.json"
+      ],
+      "sourceMaps": true,
+      "smartStep": true,
+      "outFiles": ["**/.esbuild/**/*.js"],
+      "protocol": "inspector",
+      "autoAttachChildProcesses": true,
+      "console": "integratedTerminal",
+      "outputCapture": "console"
+    },
```
Comment (adriencaccia) [inline commonConfiguration/.vscode/launch.json:34]: We can remove the inputs and the first configuration, since there are no lambdas in the repo

**Review by adriencaccia — CHANGES_REQUESTED** [review]

File: commonConfiguration/.vscode/launch.json (hunk 1 above)
Comment (fargito) [reply commonConfiguration/.vscode/launch.json:34]: This will be useful in the future, when we add the orchestrator backend

Comment (adriencaccia) [reply commonConfiguration/.vscode/launch.json:34]: fair enough

**Pull request merged**